plugins/org.jkiss.dbeaver.core/plugin.xml

```diff
@@ -519,10 +519,10 @@
 
                 <separator name="driverGroup" visible="true"/>
                 <command commandId="org.jkiss.dbeaver.core.new.connection">
-                    <visibleWhen checkEnabled="true"/>
+                    <visibleWhen checkEnabled="true"><test property="org.jkiss.dbeaver.core.global.canCreateConnection"/></visibleWhen>
```

Comment:
```suggestion
                    <visibleWhen checkEnabled="true">
                        <test property="org.jkiss.dbeaver.core.global.canCreateConnection"/>
                    </visibleWhen>
```

Reply:
I do not see any specific reasons for that.

Reply:
Readability.